### ISSUE TITLE
Training DCGAN will result in gray images

### ISSUE THREAD

**Issue by mojganaz** [junyanz/iGAN#17]
I am using train_dcgan.py to train the network over outdoor_64 images. However, after about 10 epochs of training, the generated images by the network become gray. Here is an example output file: 
![gen_00011](https://user-images.githubusercontent.com/5288525/33233290-9d00f692-d1e1-11e7-8f3d-5ec96ecf1acf.png)

Any idea on what the problem could be?

**Comment by boluoweifenda** [comment]
@mojganaz  I have the same question, I think it may be caused by some hyperparameters and generator "learns" to produce gray images to minimize it's loss function. I haven't fix the problem yet, please let me know if you have any idea, thanks!

**Comment by junyanz** [comment]
It happened sometimes for DCGAN training. I would recommend that you use the latest GAN models as backend such as [progressive-gans](https://github.com/tkarras/progressive_growing_of_gans).

**Comment by AbhishekSen** [comment]
@mojganaz Same problem here , did you find out what was the issue and how did you fix it ?

**Comment by junyanz** [comment]
DCGAN is pretty old. If you just need to train a GAN model, you can probably try more recent work such as [GAN-stability](https://github.com/LMescheder/GAN_stability) or [style-gan ](https://github.com/NVlabs/stylegan).
